docs/topics/wasm/wasm-troubleshooting.md

```diff
@@ -108,4 +108,27 @@ export { moduleExports as default };
 You can place your new `.mjs` file in the resources folder, and it will automatically be placed next to the main `.mjs` file during the build process.
 
 You can also place your `.mjs` file in a custom location. In this case, you need to either manually move it next to the main `.mjs` file or 
-adjust the path in the import statement to match its location.
\ No newline at end of file
+adjust the path in the import statement to match its location.
+
+## Slow Kotlin/Wasm compilation
+
+When working on Kotlin/Wasm projects, you may experience slow compilation times. This happens because the Kotlin/Wasm 
+toolchain recompiles the entire codebase every time you make a change.
+
+To mitigate this issue, Kotlin/Wasm targets support incremental compilation, which enables the compiler to recompile only 
+those files relevant to changes from the last compilation.
+
+Using incremental compilation reduces the compilation time. It doubles 
```

Comment:
```suggestion
Using incremental compilation reduces the compilation time and can significantly improve development speed. Currently, it doubles 
```

Reply:
we can remove "and can significantly improve development speed." if you feel like it repeats info too much. 
I think adding Currently before the start of the sentence helps to belabor the point that it's just the current setup

Reply:
Thanks for the suggestion. I think I'll leave it as it is with fewer words (: